gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -1380,6 +1474,51 @@ gum_darwin_enumerate_ranges (mach_port_t task,
   }
 }
 
+static gboolean
+gum_darwin_fill_file_mapping (gint pid,
+                              mach_vm_address_t address,
+                              GumFileMapping * file,
+                              struct proc_regionwithpathinfo * reginfo)
+{
+  gint retval, len;
+
+  retval = __proc_info (PROC_INFO_CALL_PIDINFO, pid, PROC_PIDREGIONPATHINFO,
+            (uint64_t) address,  reginfo, sizeof (struct proc_regionwithpathinfo));
+
+  if (retval == -1) {
+      return FALSE;
+  }
+
+  len = strnlen (reginfo->prp_vip.vip_path, MAXPATHLEN);
+  reginfo->prp_vip.vip_path[len] = '\0';
+
+  if (len > 0) {
```

Comment:
`{` should go on the next line.

Reply:
Actually we could replace this with an early return to avoid the block and indentation.